go.mod

```diff
@@ -3,97 +3,274 @@ module github.com/vmware-labs/distribution-tooling-for-helm
 go 1.21
 
 require (
+	github.com/DataDog/go-tuf v1.0.2-0.5.2
 	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/google/go-containerregistry v0.15.2
+	github.com/google/go-containerregistry v0.16.1
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/pterm/pterm v0.12.63
+	github.com/sigstore/cosign/v2 v2.2.1
 	github.com/sirupsen/logrus v1.9.3
-	github.com/spf13/cobra v1.7.0
+	github.com/spf13/cobra v1.8.0
 	github.com/stretchr/testify v1.8.4
 	github.com/vmware-labs/yaml-jsonpath v0.3.2
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.13.1
+	oras.land/oras-go/v2 v2.3.1
 )
 
 require (
-	github.com/Microsoft/hcsshim v0.11.0 // indirect
-	github.com/google/gnostic-models v0.6.8 // indirect
-	golang.org/x/tools v0.9.1 // indirect
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20230525234030-28d5490b6b19 // indirect
+	cloud.google.com/go/compute v1.23.2 // indirect
```

Comment:
There are quite a lot of new dependencies here. I think the pre-release version of `dt` that you sent me was quite bigger than the existing release (existing is 25Mb, pre-release is 46Mb). But I don't see that many new imports in the existing code. I see a few new uses of go-containerregistry but crane is quite small. 

Do you think the cosign and oras libraries might be somehow being propagated to the final release binary?

Reply:
I cannot see any difference in the sizes (this branch produces a 40MB binary). Maybe you checked the compressed version of the release?

Reply:
Pebkac!